lib/screens/signup/widgets/mobile_number_field.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class MobileNumber extends StatelessWidget {
```

Comment:
maintain consistency. all similar widgets' names end with Field, except for this one.
```suggestion
class MobileNumberField extends StatelessWidget {
```